.travis.yml

```diff
@@ -12,15 +13,15 @@ env:
     - COLLECTION_VERSION: 0.1.10
     - secure: "tE6GtwrRU+Kjobx/94xqR2MqM20pHCnrLcHgPzIHA3npdwuA+GjCBiBLTkEEQM4fUWIfzUTyjSr9bZErm1PTI1GcIRdniTgJ3ZzSSkE7tgeYALB/7xsusB57SlmbBQm2SGwU558uWZ3NHEsi0WTgD8GKZo77OpGX72FZKsVXOz6k2wve51sOtoSVjgCsvWTmZHx4ynGdiA5wFkZfaEcjXECahKtunW+MlB5kpJzkVeLRUEXFMhWlsIYiA5nj8OI/X3Nk9ugh1ribENX9LrjpgrqQ9YariZ8G6py1ONuKZIn2g7xs5kNQ3qL6HL6N7SoUxiwH16CfSyugFaYiMfaxQ4NUVGGRHS4vSGbNIf+gLHcYvP40miI1f/+pntCzqygZMhW73FX2o+KH2OGv09khOl8k1nDg2/XvW0kCc/FU6l+Jp5wCC8H9X2uiULtQpRqts5TzIonlPEzGIpfGFgJ5m54Emhv9gjG1Z5OOyL/qae1Wr+L/uhiFafcglZYh8NHEMWCUCkeqFqR2kDmUMtdgYLD7Q7NdwlL/PSVVs1l7UPiQHlnecQKEHN7CvR3eKByTEmkCKafRYh/JQ9rBt9sZc7aAPVu+w3wWUwbHS4o4vVnmyXvJb1PeJSiuynF7CBo4Qd6qj4YwX8gLK6PylGyaMOp169u6xw1mo5/CX0pJ3x4="
 
-matrix:
+jobs:
   include:
-    - name: "Python 3.6 - Netbox 2.5 - Latest PyPi Ansible"
+    - name: "Python 3.6 - Netbox 2.6 - Latest PyPi Ansible"
       python: 3.6
-      env: PYTHON_VER=3.6 VERSION=v2.5
+      env: PYTHON_VER=3.6 VERSION=v2.6
       install:
         - cd ..
         # Setup netbox container for integration testing
-        - git clone https://github.com/FragmentedPacket/netbox-docker.git
+        - git clone https://github.com/netbox-community/netbox-docker.git
```

Comment:
If you change this back, it should work for NetBox 2.6. There is something funky with the original repo which is why I cloned it. I believe it had to do with something crashing during startup scripts.

You can keep the other one the same for 2.7 since that appears to work properly.

Reply:
Yeah ok, that has fixed it.
FYI the reason for using the official one in 2.7 is to meet the new Redis requirements.